cookbook/wandb_logged_chain/wandb_prompts/traceprompt.py

```diff
@@ -12,8 +13,7 @@ class TracePrompt(Prompt):
     """Parent class for inherited WandB functionality."""
 
     span_type: Literal["tool", "llm"]
-
-    model_config = ConfigDict(arbitrary_types_allowed=True)
+    _call_params: OpenAICallParams = OpenAICallParams(model="gpt-3.5-turbo-1106")
```

Comment:
unfortunately you're going to have to change this again cuz its now call_params instead of_call_params